Source/Examples/WPF.SharpDX/SimpleDemo/MainViewModel.cs

```diff
@@ -40,15 +40,14 @@ public class MainViewModel : BaseViewModel
         public Color4 DirectionalLightColor { get; private set; }
         public Color4 AmbientLightColor { get; private set; }
 
-
         public MainViewModel()
         {
             // titles
-            this.Title = "Simple Demo";

```

Comment:
In the WPF (not SharpDX) I am strictly enforcing StyleCop:
http://stylecop.soyuz5.com/SA1101.html
:-)